indico/web/forms/fields.py

```diff
@@ -343,13 +343,17 @@ def _get_event_person_for_user(self, user):
     def _get_event_person(self, data):
         person_type = data.get('_type')
         if person_type is None:
-            email = data['email'].lower()
-            user = User.find_first(~User.is_deleted, User.all_emails.contains(email))
-            if user:
-                return self._get_event_person_for_user(user)
-            else:
-                person = self.event.persons.filter_by(email=email).first()
-                return person or self._create_event_person(data)
+            if 'email' in data:
```

Comment:
`if data.get('email')` in case we end up sending it empty instead of omitting it in the future?